test.js

```diff
@@ -133,6 +133,11 @@ test('leading underscore', t => {
 	t.is(slugify('____-___foo__bar', {preserveLeadingUnderscore: true}), '_foo-bar');
 });
 
+test('trailing dash', t => {
+	t.is(slugify('foo bar-', {preserveTrailingDash: true}), 'foo-bar-');
+	t.is(slugify('foo-bar--', {preserveTrailingDash: true}), 'foo-bar-');
```

Comment:
Add some more assertions. For example, `foo-bar -`, `foo-bar `, etc.

Reply:
https://github.com/sindresorhus/slugify/pull/57/commits/c0b0ef75102bc35baaf82e83979c192eb3ac1432